css/src/components/popover.scss

```diff
@@ -0,0 +1,35 @@
+.popover {
+	display: inline-block;
+	position: relative;
+
+	.popover-summary {
+		color: $hyperlink;
+		list-style: none;
+		cursor: pointer;
+	}
+
+	.popover-content {
+		position: absolute;
+		min-width: 150px;
```

Comment:
1. extract to variable at top of file? `$popover-min-width: 160px;`
2. 160px? makes it more divisible by 4s and 8s and 16s.

Reply:
It was a random number tbh cause I didn't have any design when was writing this. I'll update according to the file Courtny will provide and your recommendation about divisibility by 4/8/16.

Reply:
Updated to 224px. Matches the design and divisible by 4/8/16